### PR TITLE
release 4.0.1

### PR DESCRIPTION
**Pull request by mikemorency**
##### SUMMARY
There was an issue uploading release 4.0.0 to RedHat Automation Hub, so they've asked us to create a new release.

There is no functionality difference between 4.0.0 and 4.0.1.

##### ISSUE TYPE
- Release

### CODE REVIEW

**Review by mariolenz — CHANGES_REQUESTED** [review]
A minor change means we need a minor release (4.1.0), a bugfix release (4.0.1) would break semantic versioning ;-)

I see two possibilities:

1. Consider if we can categorize #524 as a bugfix (because the scenario guides are old and buggy) and update the two changelog files accordingly
2. Prepare 4.1.0 instead of 4.0.1

I _tend_ to 2 but am not 100% sure.

**Comment by mikemorency** [comment]
hmmm ok. technically documentation changes dont even need a changelog, so i think keeping this as a bugfix release instead of a minor release would be more accurate. ill update the changelogs from 524

File: CHANGELOG.rst
Comment (mariolenz) [inline CHANGELOG.rst:11]:
```suggestion
Bugfixes
--------
```

It's a little bit weird if `changelog/changelog.yaml` says it's a bugfix and `CHANGELOG.rst` says it's a minor change ;-)

Apart from this LGTM!

**Comment by mikemorency** [comment]
@mariolenz can you approve/remove the change request if it looks good? I think its blocking the merge automation

**Review by mariolenz — APPROVED** [review]
LGTM

**Pull request merged**
